indra/sources/phosphoelm/processor.py

```diff
@@ -0,0 +1,139 @@
+import logging
+import requests
+
+from indra.statements import Phosphorylation, Evidence, Agent
+
+gilda_url = 'http://grounding.indra.bio/ground'
+logger = logging.getLogger(__file__)
+
+
+def _gilda_grounder(entity_str):
+    # Try to find a namespace for the enzyme entity string, return the string
+    # that provided the match
+    res = requests.post(gilda_url, json={'text': entity_str})
+    if res.status_code == 200 and res.json():
+        db_ns = res.json()[0]['term']['db']
+        db_id = res.json()[0]['term']['id']
+        return entity_str, db_ns, db_id
+    else:
+        if res.status_code != 200:
+            logger.warning('Gilda service responded with status code %d' %
+                           res.status_code)
+        return entity_str, None, None
+
+
+class PhosphoElmProcessor(object):
+    def __init__(self, file_dump_json=None, keep_empty=False):
```

Comment:
Can file_dump_json really be None? What does the processor do if that's the case?